src/test/regress/expected/isolation_acquire_distributed_locks.out

```diff
@@ -0,0 +1,1031 @@
+Parsed test spec with 3 sessions
```

Comment:
amazing isolation test coverage, thanks!

I think there is one more test I'd love to see is we directly/only lock citus_local_table  (not the view as well) and show that the workers are blocked.